app/src/main/java/org/fossasia/susi/ai/rest/responses/others/RelatedWebTopics.kt

```diff
@@ -0,0 +1,25 @@
+package org.fossasia.susi.ai.rest.responses.others
+
+
+/**
+ * <h1>Kotlin Data class to parse related topics object in retrofit response from websearch client.</h1>
+ *
+ * Created by mayank on 09-12-2016.
+ */
+
+class RelatedTopics (
+
+    var url: String? = null,
+    var text: String? = null,
+    var icon: WebIcon? = null,
+    var result: String? = null
+)
+
+class WebIcon ( var url: String? = null )
+
+class WebSearch (
+
+        var heading: String? = null,
+        var relatedTopics: List<RelatedTopics>? = null
+
+)
```

Comment:
@S2606 Nice work here. Similarly, make 2 more files: one containing response from location. Other containing responses from susi. So, we will finally have one file corresponding to one client i.e Location client, SUSI, client and websearch client.